content/doc/administrate/activity.md

```diff
@@ -0,0 +1,38 @@
+---
+type: docs
+title: Activity
+description: Applications activity lifecycle explained
+tags:
+- lifecycle
+keywords:
+- activity
+- unreachable
+- timeout
+- diskfull
+- scaling
+- maintenance
+type: docs
+---
+
+Follow your application activities and lifecycle from the **Activity** option in the application menu. Every time your application restarts, a new block appears on the timeline. Here's what you might see on the timeline:
+
+![Activity timeline](/images/doc/activity.png)
+
+Each block indicates if the deployment was successful or unsuccessful, along with the following information:
+
+- **Date**: when the activity happened and from where (`Console`, `Git`, `Github` or `Monitoring`)
+- **Author**: which member of your organization triggered the activity
+- **Commit ID**: on which commit the activity happened
+- **Redeploy this commit**: the option to redeploy a previous commit (useful to quickly roll-back if needed)
+
+## Re-deployments triggered by the monitoring
+
+If the monitoring triggers a redeployment, you'll see no author and the message explaining the reason, like `Monitoring/Unreachable`. The messages you might get are the following:
```

Comment:
Maybe add a hint here or later on how to debug application ? like get a look on metrics in grafana :thinking:

Reply:
```suggestion
If the monitoring triggers a re-deployment, you'll see no author and the message explaining the reason, like `Monitoring/Unreachable`. The messages you might get are the following:
```